### PR TITLE
Tbcm 37 activities bundles tags

### PR DESCRIPTION
**Pull request by PeterPavlatos**
- Added tags component to right side bundle list
- Added code to be able to access and got to next page
-  Added search component to left and right side columns

### CODE REVIEW

File: apps/web/src/components/RightSideBarActivities.tsx [hunk 1]
```diff
@@ -80,8 +76,19 @@ export const RightSideBarActivites: React.FC = () => {
...
               filteredData.map((item: any) => {
                 return (
-                  <div key={item.id}>
-                    <Checkbox name='checked' value={item.id} label={item.name} />
+                  <div key={item.id} className='flex flex-1 items-center'>
+                    <div className='flex-initial w-5/6'>
+                      <Checkbox
+                        name='checked'
+                        value={item.id}
+                        styles='text-bcDarkBlue accent-bcBlueLink'
```
Comment (karthikktamilmani-freshworks) [inline apps/web/src/components/RightSideBarActivities.tsx:84]: I don't see this being blue color in the design.

File: apps/web/src/components/planning/CareActivitiesBundle.tsx [hunk 2]
```diff
@@ -2,24 +2,54 @@ import { PageTitle } from '@components';
...
-  step: number;
+  step?: number;
   title: string;
 }
 export const initialValues = {
   careActivities: [],
   careActivityBundle: [],
 };
 
+const CareActivitiesForm: React.FC<CareActivitiesBundleProps> = ({ title }) => {
```
Comment (karthikktamilmani-freshworks) [inline apps/web/src/components/planning/CareActivitiesBundle.tsx:19]: Kindly rebase the PR

Comment (PeterPavlatos) [reply apps/web/src/components/planning/CareActivitiesBundle.tsx]: ok

File: apps/api/src/database/database.module.ts [hunk 3]
```diff
@@ -47,8 +47,8 @@ const environmentSpecificConfig = getEnvironmentSpecificConfig(nodeEnv);
 const appOrmConfig: PostgresConnectionOptions = {
   ...config,
   ...environmentSpecificConfig,
-  migrationsRun: true,
-  synchronize: false,
+  migrationsRun: false,
+  synchronize: true,
```
Comment (karthikktamilmani-freshworks) [inline apps/api/src/database/database.module.ts:51]: Not required.

File: apps/web/src/components/RightSideBarActivities.tsx [hunk 4]
```diff
@@ -82,8 +84,14 @@ export const RightSideBarActivites: React.FC = () => {
               filteredData &&
               filteredData.map((item: any) => {
                 return (
-                  <div key={item.id}>
-                    <Checkbox name='careActivities' value={item.id} label={item.name} />
+                  <div key={item.id} className='flex flex-1 items-center'>
+                    <div className='flex-initial w-5/6'>
+                      <Checkbox name='checked' value={item.id} label={item.name} />
```
Comment (karthikktamilmani-freshworks) [inline apps/web/src/components/RightSideBarActivities.tsx:89]: I think I've updated the name to careActivities. Kindly update and test the flow

File: apps/web/src/components/RightSideBarActivities.tsx [hunk 5]
```diff
@@ -21,6 +24,8 @@ export const RightSideBarActivites: React.FC = () => {
       setItems(newCareActivity);
       //Set checked ids to previously selected ids when care activity changes
       values.careActivities = _.filter(values.careActivityBundle[values.careActivityID]);
+      //Empty array when careActivityID changes
+      values.checked = [];
```
Comment (karthikktamilmani-freshworks) [inline apps/web/src/components/RightSideBarActivities.tsx:28]: I think I've updated the name to careActivities. Kindly update and test the flow

Comment (PeterPavlatos) [reply apps/web/src/components/RightSideBarActivities.tsx:28]: Updated to current state.

File: apps/web/src/components/RightSideBarActivities.tsx (hunk 4 above)
Comment (PeterPavlatos) [reply apps/web/src/components/RightSideBarActivities.tsx:89]: Updated to current develop

**Comment by karthikktamilmani-freshworks** [comment]
This is merged with TBCM-44

**Pull request closed without merging**